src/multi_select.coffee

```diff
@@ -15,12 +15,24 @@ Ember.Widgets.MultiSelectOptionView = Ember.View.extend
     @notifyPropertyChange 'label'
   , 'context', 'labelPath'
 
+Ember.Widgets.MultiSelectOptionView = Ember.Widgets.SelectOptionView.extend
+  processDropDownShown: ->
+    @_super()
+    @get('controller').focusTextField()
```

Comment:
A controller should not manipulate the DOM, it breaks the separation of concerns